src/ASM/LR/ASMu2D.C

```diff
@@ -2066,6 +2066,8 @@ bool ASMu2D::evalSolution (Matrix& sField, const Vector& locSol,
     fe.v = gpar[1][i];
     int iel = lrspline->getElementContaining(fe.u,fe.v);
 
+    const LR::Element* el = lrspline->getElement(iel);
+
```

Comment:
Should perhaps move this to after the `if (iel != lel)` check, just in case `lrspline->getElement()` does not handle an invalid argument gracefully.